srp/src/client.rs

```diff
@@ -105,6 +124,14 @@ pub struct SrpClientVerifier<D: Digest> {
     key: Vec<u8>,
 }
 
+/// RFC 5054 SRP client state after handshake with the server.
+pub struct SrpClientVerifierRfc5054<D: Digest> {
+    m1: Output<D>,
+    m2: Output<D>,
+    key: Vec<u8>,
+    session_key: Vec<u8>,
```

Comment:
I think you can use `D::Output` for `session_key`.

Reply:
Yes, I think we can use `D::Output` as long as we have a getter method that returns this in a more primitive type, because at least in my use-case, this session key is used for every encryption after the key agreement.
It is nicer to hide away all these types used internally before the output